src/components/Greeting.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function Greeting() {
+  const greetings = useSelector((state) => state.greetingReducer.greeting);
+
+  return (
+    <div>
+      <h2>THESE ARE RANDOM GREETINGS</h2>
+      <ul>
+        <li>{greetings.message}</li>
```

Comment:
- [ ] I think there is an error here in your front end project so kindly make sure to solve it I can show an image about the error that I see so kindly work on it to solve the error 🚀💯💯

![image](https://user-images.githubusercontent.com/58553711/204414484-0b54f500-89fc-45aa-9e3d-bec8e74d3b92.png)